pkgs/development/compilers/ghc/8.4.2.nix

```diff
@@ -24,7 +24,15 @@
   # platform). Static libs are always built.
   enableShared ? !targetPlatform.useAndroidPrebuilt
```

Comment:
You wish to allow dynamic linking here but not 8.2?

Reply:
I don't follow? That line wasn't even changed?

Reply:
Ahh... ok. fixed.